src/Pane.java

```diff
@@ -3,60 +3,59 @@
-    final static int width = 640;
-    final static int height = 480;
-    Button one_player;
-    Button two_players;
+    final static int WIDTH = 640;
+    final static int HEIGHT = 480;
+    Button onePlayer;
+    Button twoPlayers;
     Button restart;
     Label label;
     JButton[][] fields;
-    int[] x_coordinates = new int[3];
-    int[] y_coordinates = new int[3];
-    final static Color player1_color = Color.orange;
-    final static Color player2_color = new Color(0, 150, 200);
-    final static Color game_over_field_color = Color.lightGray;
+    int[] xCoordinates = new int[3];
+    int[] yCoordinates = new int[3];
+    final static Color player1Color = Color.orange;
+    final static Color player2Color = new Color(0, 150, 200);
+    final static Color gameOverFieldColor = Color.lightGray;
     // First panel constructor
     Pane(){
-        standard_settings();
-        one_player = new Button("One player");
-        one_player.setLocation((int)(width-2*one_player.getWidth())/3, (int)(height-one_player.getHeight())/2);
-        one_player.setOpaque(true);
-        this.add(one_player);
+        standardSettings();
+        onePlayer = new Button("One player");
+        onePlayer.setLocation((int)(WIDTH-2*onePlayer.getWidth())/3, (int)(HEIGHT-onePlayer.getHeight())/2);
+        onePlayer.setOpaque(true);
+        this.add(onePlayer);
 
-        two_players = new Button("Two players");
-        two_players.setLocation((int)(width-2*two_players.getWidth())/3*2+two_players.getWidth(), (int)(height-two_players.getHeight())/2);
-        two_players.setOpaque(true);
-        this.add(two_players);
+        twoPlayers = new Button("Two players");
+        twoPlayers.setLocation((int)(WIDTH-2*twoPlayers.getWidth())/3*2+twoPlayers.getWidth(), (int)(HEIGHT-twoPlayers.getHeight())/2);
+        twoPlayers.setOpaque(true);
+        this.add(twoPlayers);
 
         label = new Label("Welcome to TicTacToe!");
-        label.setLocation((int)(width-label.getWidth())/2, ((int)(height-3*120)/2-label.getHeight())/2);
+        label.setLocation((int)(WIDTH-label.getWidth())/2, ((int)(HEIGHT-3*120)/2-label.getHeight())/2);
         this.add(label);
     }
     // Second panel constructor
-    Pane(boolean whose_turn, boolean is_running, boolean draw){
-        standard_settings();
+    Pane(boolean whoseTurn, boolean isRunning, boolean draw){
+        standardSettings();
         restart = new Button("Restart");
-        restart.setLocation((int)(width-restart.getWidth())/2, (int)(height-restart.getHeight())/2);
+        restart.setLocation((int)(WIDTH-restart.getWidth())/2, (int)(HEIGHT-restart.getHeight())/2);
         restart.setOpaque(true);
-//        this.add(restart, Integer.valueOf(0));
 
-        for(int i=0; i<x_coordinates.length; i++){
-            x_coordinates[i] = (int)(width-3*120)/2 + i*120;
+        for(int i=0; i<xCoordinates.length; i++){
+            xCoordinates[i] = (int)(WIDTH-3*120)/2 + i*120;
```

Comment:
co oznacza 120? dobrze jest unikać takich zapisów i wrzucać je tak jak zrobiłaś to z WIDTH i HEIGHT (potem w 1 miejscu zmienisz i się przeskaluje)